CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# [MongooseIM 4.0.0](https://github.com/esl/MongooseIM/releases/tag/4.0.0) - 2020-09-30
+## Overview
+MongooseIM 4.0 is all about putting a friendly face to MongooseIM’s amazing features.
+This includes the addition of an Erlang agnostic configuration allowing a broader pool of developers to benefit from MongooseIM.
+Logging and Kubernetes improvements to be more DevOps friendly and Manager friendly with load testing enabling managers to see the benefits of MongooseIM easily.
+
+## All changes
+
+### Added
+* Agnostic configuration file: MongooseIM can now be configured with an entirely revamped [TOML](https://github.com/toml-lang/toml) configuration file (#2801)
+  * Pesky bugs have been fixed
+  * Implementation details have been abstracted away
+  * Documentation has been improved
+* Structured logging: the main idea of structured logging is that if we have properties of an event, let’s log them as they are, without losing their structure, and format it in a way that is most appropriate for the system we’re sending them to.
+  * Moved away from [lager](https://github.com/erlang-lager/lager), towards [OTP logger](https://erlang.org/doc/man/logger.html) (#2810)
```

Comment:
Is the comma necessary?

Reply:
I think so, those are two different sentences 🤔 @goddammit ?